index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>THIERRY'S RESTAURANT</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-ul">
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/phone-view-icons/facebook.png"
+                  alt="facebookIcon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/phone-view-icons/twitter.png" alt="twitter"
+              /></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="https://www.google.com/">Log out</a></li>
+          </ul>
+        </div>
+        <div class="mobile-nav">
+          <img
+            src="./images/phone-view-icons/hamburger.svg"
+            alt="menu-icons"
+            id="menu-icon"
+          />
+        </div>
+        <div class="main-menu-container">
+          <div class="main-menu" id="main-menu">
+            <span class="chef-icon"
+              ><a href="index.html"><img
+                src="./images/phone-view-icons/download.png"
+                alt="chef-icon"
+            /></a></span>
+            <span class="close"
+              ><img
+                src="./images/phone-view-icons/x.svg"
+                alt="close-icon"
+                id="close-icon"
+            /></span>
+            <ul class="menu-list">
+              <li><a href="About.html" id="about">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#" class="button" id="button">CC Campaign</a></li>
+            </ul>
```

Comment:
- [ Optional ] I think it would be better if you follow the design provided by Behance as much as possible. On the Behance design, this menu is `aligned` to the `right` while on your design it's `centered`. The design provided by Microverse for this project represents what a client may present to you to develop and thus following it, shows just how keen you are when implementing a client's request. I highly recommend that you implement this change.

## Your design
![Screenshot from 2023-07-06 22-47-08](https://github.com/axcell-0/first-capstone/assets/16392046/c2fcee3e-96e3-47a7-a163-3b2a8737d0b0)

## Behance design
![Screenshot from 2023-07-06 23-00-51](https://github.com/axcell-0/first-capstone/assets/16392046/a4a6c5ab-790a-49ab-8a91-09136fc7ebbf)